src/main/java/menion/android/whereyougo/utils/NotificationService.java

```diff
@@ -87,18 +87,23 @@ private void startNotificationService(boolean background) {
         if (runing && background == foreground) {
            if (foreground) {
               stopForeground(true);
+              runing = false;
            } else {
               mNM.cancel(notification_id);
+              runing = false;
```

Comment:
better set runing outside the if/then statement.

Reply:
moved the the set outside of the if/then Statement.